test/Controller/PageTestCase.php

```diff
@@ -85,6 +85,7 @@ final public function it_has_global_metadata()
      */
     public function it_has_cache_headers()
     {
+        $this->markTestSkipped('This test can be skipped for reviewed-preprint launch');
```

Comment:
The message should be:

```
// @todo - this can be switched on again after https://github.com/elifesciences/issues/issues/7764
$this->markTestSkipped('This test can be skipped as non-deterministic!');
```